config/controllers.config.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Zend\ServiceManager\ServiceLocatorInterface;
+use ZfcUserAdmin\Controller\UserAdminController;
+
+return array(
+    'controllers' => array(
+        'factories' => function (ServiceLocatorInterface $sm) {
+            $createUserForm = $sm->get('zfcuseradmin_createuser_form');
+            $editUserForm = $sm->get('zfcuseradmin_edituser_form');
+            $options = $sm->get('zfcuseradmin_module_options');
+            $userMapper = $sm->get('zfcuser_user_mapper');
+            $adminUserService = $sm->get('zfcuseradmin_user_service');
+            $zfcUserOptions = $sm->get('zfcuser_module_options');
+
+            return new UserAdminController(
+                $createUserForm,
+                $editUserForm,
+                $options,
+                $userMapper,
+                $adminUserService,
+                $zfcUserOptions
+            );
```

Comment:
Cant u put this in a separate factory class?
This makes that this line is also cachable and can make it faster.